models/fates/R/write.configs.FATES.R

```diff
@@ -19,24 +19,40 @@
 ##' @return none
 ##' @export
 ##' @author Mike Dietze, Shawn Serbin
-##-------------------------------------------------------------------------------------------------#
+##-------------------------------------------------------------------------------------------------# # nolint
+
+# example trait.values list
+trait.values <- list()
+trait.values$temperate.coniferous <- list()
+trait.values$env <- list()
+
+tc_traits <- as.data.frame(matrix(c(9999, 8888), nrow=1, ncol=2))
+colnames(tc_traits) <- c("leaf_turnover_rate", "veg_respiration_Q10")
+# Fates names for these params are "fates_leaf_long", "q10_mr"
+
+trait.values$temperate.coniferous <- tc_traits
+settings <- '/pecan/models/fates/R/test.fates.xml'
+run.id <- 'test_oneyear'
+
+# example trait.values list
+
 write.config.FATES <- function(defaults, trait.values, settings, run.id){
 
    ## site information
    site <- settings$run$site
-   site.id <- as.numeric(site$id)
+   site.id <- site$id # change1: 772 -> SOD1
   
    # find out where things are
    local.rundir <- file.path(settings$rundir, run.id) ## this is on local machine for staging
    rundir       <- file.path(settings$host$rundir, run.id)  ## this is on remote machine for execution
-   casedir      <- file.path(rundir,"case") 
+   casedir      <- file.path(rundir,run.id) ## /ctsm-api/resources/cases/case_SOD1, change2: 'case' -> site.id
    outdir       <- file.path(settings$host$outdir, run.id)
-   refcase      <- settings$model$binary
+   refcase      <- settings$model$binary # question1: refcase==casedir?
    bld          <- file.path(refcase,"bld")
    binary       <- file.path(bld,"cesm.exe")
-   indir        <- file.path(rundir,"input") ## input directory
+   indir        <- file.path(rundir,"data/shared") ## input directory
    default      <- settings$run$inputs$default$path ## reference inputs file structure
-   site_name    <- paste0(site.id %/% 1000000000, "-", site.id %% 1000000000)
+   site_name    <- paste0(site.id, "_c", run.id) ## change3 
```

Comment:
I'm a bit concerned about the logic change here and on the casedir line. Could you explain what you're intending? Be aware that changes of internal logic can't change the nature of what PEcAn is passing into each function.